SofascoreAcademy/SofascoreAcademy/Views/MainViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  MainViewController.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 08.04.2024..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+protocol parentSportSlugPicker {
+    func getPressedTab(request: sport_slug?)
+}
+
+protocol settingsButtonTap {
+    func reactToSetingsTap()
+}
+
+protocol tableCellTap {
+    func reactToCellTap(match: matchData)
+}
+
+enum sport_slug {
```

Comment:
```suggestion
enum SportSlug {
```

nor ovako nesto bi bio bolji naming